src/runtime_src/core/tools/common/tests/aie_pl_util/pl_controller_aie2.cpp

```diff
@@ -267,52 +238,5 @@ std::vector<tile_type> plController_aie2::get_tiles(const std::string& graph_nam
     return tiles;
 }
 
-std::vector<buffer_type> plController_aie2::get_buffers(const std::string& port_name) {
```

Comment:
Uses reference to the dma_lock_report.json file which is not relevant for aie2 pl test.